_data/simple-icons.json

```diff
@@ -6927,8 +6927,8 @@
         },
         {
             "title": "Wattpad",
-            "hex": "F68D12",
-            "source": "https://www.wattpad.com/press/#assets"
+            "hex": "FF500A",
+            "source": "https://www.wattpad.com"
```

Comment:
Worth updating source to `https://company.wattpad.com/media-kit`?

Reply:
Done. Thanks for your input! 😊